### PR TITLE
Updated scruffy versions

### PR DESCRIPTION
**Pull request by sksamuel**

### CODE REVIEW

**Comment by hamiltont** [comment]
If you have trouble debugging the compile errors through travis-ci, you can easily spin up a local development [virtual machine](https://frameworkbenchmarks.readthedocs.org/en/latest/Development/Installation-Guide/#vagrant-development-environment)

**Comment by sksamuel** [comment]
Thanks for the tip @hamiltont luckily I know what this error is :)

**Comment by sksamuel** [comment]
@hamiltont should be good to go now :)

**Comment by mcocciaTE** [comment]
Everything looks good to me!

File: frameworks/Scala/scruffy/src/test/scala/BenchmarkVerify.scala [hunk 1]
```diff
@@ -11,7 +11,7 @@ object BenchmarkVerify extends App {
 
   val sw = Stopwatch()
 
-  val futures = for ( k <- 1 to 10000 ) yield {
+  val futures = for ( k <- 1 to 30000 ) yield {
```
Comment (msmith-techempower) [inline frameworks/Scala/scruffy/src/test/scala/BenchmarkVerify.scala]: What is this I don't even...

Comment (sksamuel) [inline frameworks/Scala/scruffy/src/test/scala/BenchmarkVerify.scala]: Just creating 30,000 deferred requests to verify the endpoints work as expected when a change is made. It's code in the src/test source folder so isn't include in the JAR that is built when the build script is invoked.

Comment (msmith-techempower) [inline frameworks/Scala/scruffy/src/test/scala/BenchmarkVerify.scala]: So, this code is not executed during an actual benchmark run?

Comment (sksamuel) [inline frameworks/Scala/scruffy/src/test/scala/BenchmarkVerify.scala]: Correct. It's not even included in the package that is built when you build the jar. It's just a smoke test I run when I upgrade the version of Scruffy used by the tests. It's typical in Java/Scala to keep the test code in a source folder called src/test and the main code in src/main.

**Comment by msmith-techempower** [comment]
The let that which must be done be done!

**Pull request merged**